README.md

```diff
@@ -159,6 +157,88 @@ platform](https://github.com/asreview/systematic-review-datasets).
 asreview data dedup benchmark:van_de_schoot_2017 -o van_de_schoot_2017_dedup.csv
 ```
 
+### Data Compose (Experimental)
+Compose is where datasets with different labels (or no labels) can be assembled into a single dataset.
+
+❗ Compose is an experimental feature. We would love to hear your feedback. Please keep in mind that this feature can change in the future. 
+
+#### Data format
+Your data files need to be in tabular file format or RIS file format.
+The output file and all input files should be in the same format.
+
+- **Tabular file format:**
+Supported tabular file formats are `.csv`, `.tab`, `.tsv` or `.xlsx`.
+Make sure your column names adhere to the predetermined set of [accepted column names](https://asreview.readthedocs.io/en/latest/data_format.html).
+
+
+- **RIS file format:**
+RIS file formats have `.ris` or `.txt` as extension and are used by digital libraries.
+Read [how to format](https://asreview.readthedocs.io/en/latest/data_format.html) your RIS files.
+ASReview converts the labeling decisions in RIS files to a binary variable: irrelevant as `0` and relevant as `1`.
+
+Records marked as unseen or with missing labeling decisions, are converted to `-1` by ASReview.
+
+#### Run script
+Assume you have `MY_DATASET_1.ris` from which you want to keep all existing labels
+and `MY_DATASET_2.ris` which you want to mark as unlabeled. Compose into a single dataset:
```

Comment:
@Rensvandeschoot I chose "mark" on purpose because you can also choose to have them unlabeled even though these records are already labeled in the input dataset `MY_DATASET_2.ris`.